ui/dialogs/albuminfodialog.go

```diff
@@ -0,0 +1,118 @@
+package dialogs
+
+import (
+	"fmt"
+	"image"
+	"net/url"
+	"strings"
+
+	"github.com/dweymouth/supersonic/backend/mediaprovider"
+	"github.com/dweymouth/supersonic/ui/layouts"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/canvas"
+	"fyne.io/fyne/v2/container"
+
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+const musicBrainzReleaseUrl = "https://musicbrainz.org/release"
+
+type AlbumInfoDialog struct {
+	widget.BaseWidget
+
+	OnDismiss func()
+
+	content fyne.CanvasObject
+}
+
+func NewAlbumInfoDialog(albumInfo *mediaprovider.AlbumInfo, albumName string, albumCover image.Image) *AlbumInfoDialog {
+	a := &AlbumInfoDialog{}
+	a.ExtendBaseWidget(a)
+
+	a.content = container.NewVBox(
+		a.buildMainContainer(albumInfo, albumName, albumCover),
+		widget.NewSeparator(),
+		container.NewHBox(
+			layout.NewSpacer(),
+			widget.NewButton("Close", func() {
+				if a.OnDismiss != nil {
+					a.OnDismiss()
+				}
+			}),
+		),
+	)
+	a.content.Resize(a.MinSize())
+	return a
+}
+
+func (a *AlbumInfoDialog) MinSize() fyne.Size {
+	return fyne.NewSize(550, a.BaseWidget.MinSize().Height)
+}
+
+func (a *AlbumInfoDialog) buildMainContainer(albumInfo *mediaprovider.AlbumInfo, albumName string, albumCover image.Image) *fyne.Container {
+	iconImage := canvas.NewImageFromImage(albumCover)
+	iconImage.FillMode = canvas.ImageFillContain
+	iconImage.SetMinSize(fyne.NewSize(100, 100))
+	title := widget.NewRichTextWithText(albumName)
+	title.Segments[0].(*widget.TextSegment).Style.TextStyle.Bold = true
+	title.Segments[0].(*widget.TextSegment).Style.SizeName = theme.SizeNameSubHeadingText
+	title.Segments[0].(*widget.TextSegment).Style.Alignment = fyne.TextAlignCenter
+
+	infoContent := widget.NewLabel("Album info not available")
+
+	if albumInfo.Notes != "" {
+		infoContent = a.infoLabel(albumInfo.Notes)
+	}
+
+	urlContainer := a.buildUrlContainer(albumInfo.LastFmUrl, albumInfo.MusicBrainzID)
+
+	return container.NewVBox(
```

Comment:
I would wrap this Vbox in another container with `layouts.MaxPadLayout` with some padding values that look good to you to add a bit of margin around the main content. But this looks really nice - thanks!